start.yml

```diff
@@ -270,11 +270,28 @@
         state: started
         msg: "the TiKV port {{ tikv_port }} is not up"
 
-    - name: wait until TiKV process is up
-      wait_for_pid: |
-        pid_file={{ deploy_dir }}/status/tikv.pid timeout=300
-        thread_name_regex='tikv-server' state=present
-      when: deployment_method == 'binary'
+    - name: wait until the TiKV status page is available
+      uri:
+        url: "http://{{ ansible_host }}:{{ tikv_status_port }}/status"
+        return_content: yes
+      register: tikv_http_result
+      until: tikv_http_result.status == 200
```

Comment:
Does the status page include any keyword that indicates the tikv instances is healthy or not?